arviz/plots/kdeplot.py

```diff
@@ -261,6 +272,27 @@ def plot_kde(
         gridsize = (128, 128) if contour else (256, 256)
         density, xmin, xmax, ymin, ymax = _fast_kde_2d(values, values2, gridsize=gridsize)
 
+        if hdi_probs is not None:
+            # Check hdi probs are within bounds (0, 1)
+            if min(hdi_probs) <= 0 or max(hdi_probs) >= 1:
+                raise ValueError("Highest density interval probabilities must be between 0 and 1")
+
+            # Calculate contour levels and sort for matplotlib
+            contour_levels = _find_hdi_contours(density, hdi_probs)
+            contour_levels.sort()
+
+            contour_level_list = [0] + list(contour_levels) + [density.max()]
+
+            # Add keyword arguments to contour, contourf
+            if contour_kwargs is None:
+                contour_kwargs = {"levels": contour_level_list}
+            else:
+                contour_kwargs.setdefault("levels", contour_level_list)
```

Comment:
Shouldn't `hdi_probs` have preference over the `levels` in `kwargs`?

Reply:
I've set it up so that it warns the user if both are provided, and then uses hdi_probs preferentially; the update is in 5595173, with unit tests in cf9be3f.